README.md

```diff
@@ -1,6 +1,6 @@
 [![Stories in Ready](https://badge.waffle.io/sw360/sw360portal.png?label=ready&title=Ready)](https://waffle.io/sw360/sw360portal)
```

Comment:
We should also update or remove the waffle badge.

Reply:
yep, i saw this configuration.
We should talk about this PR in our dev talk to conclude all changes.